APIAssignment/src/test/java/APIAssignments/BaseClass.java

```diff
@@ -24,6 +31,7 @@ public BaseClass(String productId, String productName, String productPrice, Stri
 
     @BeforeTest
```

Comment:
remove before annotation and use it in step definition class

Reply:
You already are calling this method in @Before method, so you can remove @BeforeTest annotation from here